debian/control

```diff
@@ -0,0 +1,84 @@
+Source: geopm-runtime
+Section: utils
+Priority: optional
+Maintainer: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>
+Build-Depends: debhelper-compat (=13),
+               build-essential,
+               debhelper (>= 11~),
+               dh-python,
+               elfutils,
+               libgeopmd-dev,
+               libelf-dev,
+               openssh-client,
+               python3,
+               python3-all,
+               python3-dev,
+               python3-setuptools,
+               unzip
+Standards-Version: 4.1.4
+Homepage: https://geopm.github.io
+Vcs-Git: https://github.com/geopm/geopm.git
+Vcs-Browser: https://github.com/geopm/geopm.git
+
+Package: geopm-runtime
+Section: utils
+Architecture: any
+Depends: ${misc:Depends},
+         ${shlibs:Depends},
+         libgeopm1 (= ${binary:Version}),
+         libgeopmd1 (= ${binary:Version})
+Description: The geopm runtime package provides many built-in
+ features.  An advanced use case is dynamically coordinating hardware
+ settings across all compute nodes used by an distributed HPC
+ application in response to the application's behavior and resource
+ manager requests.  The dynamic coordination is implemented as a
```

Comment:
The rules file disables MPI, so I wonder if we should avoid discussing MPI/HPC use cases here?  Also the hierarchy tree?